_posts/2022-06-30-trino-summit-call-for-speakers.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Announcing the 2022 Trino Summit"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+We are pleased to announce the upcoming 2022 Trino Summit. The summit is
+scheduled as *hybrid* event on the 10th of November 2022. You will be able to
+join us online, or you can make the trip to San Francisco and meet us at the
+Commonwealth Club on the downtown waterfront. Please be aware that you need to
+register regardless of how you'll be joining us.
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-pink" href="https://www.starburst.io/info/trinosummit/">
+        Register for the event
+    </a>
+    <a class="btn btn-orange" href="https://sessionize.com/trino-summit-2022/">
+        Sign up to speak
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Starburst is the lead sponsor for the summit, but they're happy to hear from and
+collaborate with other sponsors. If that interests you or your employer, you
+should [reach out to Starburst by email.](mailto:events@starburst.io)
+
+<!--more-->
+
+More exciting, if you'd like to give a talk at this year's Trino summit,
+we're putting out a call for speakers. We will be accepting submissions from now
+until September 15th, but we recommend submitting soon, because slots are
+filling up fast.
+
+We're looking for intermediate to advanced-level talks on a variety of themes.
+If you have an interesting story about how you were able to leverage Trino,
+found a neat way to extend it with a custom plugin, or swapped to Trino for a
+performance win, we'd love to hear about it. We're excited to expand our speaker
+lineup with talks from the broader Trino community. If you're interested, you
+can check out the speaker registration page for more information.
+
+And of course, we're looking forward to seeing you there, whether in-person or
+online!
```

Comment:
maybe do something like

So don't forget: 

and then have the two buttons again ..